TreeTracker/UI/Views/Home/HomeViewController.swift

```diff
@@ -84,6 +84,12 @@ class HomeViewController: UIViewController, AlertPresenting {
     override func viewDidLoad() {
         super.viewDidLoad()
         viewModel?.fetchTrees()
+        self.showLogoutButton()
+    }
+
+    private func showLogoutButton() {
```

Comment:
For consistency with the other ViewControllers can we create a private extension for this similar to the button actions? I'm sure as time goes on we will have more private functions we will want to drop in here.
 e.g.

```
// MARK: - Private 
private extension HomeViewController { 
    func showLogoutButton() { ... }
}
```